### ISSUE TITLE
Beim Runterfahren bleibt der Playground oft in "Exiting UI" hängen

### ISSUE THREAD

**Issue by nonlinear-labs** [nonlinear-labs-dev/C15#1080]
Der Supervisor schaltet dann nach 30 sec hart ab.

Das Problem tritt besonders häufig bei neuen Geräten nach dem Einspielen eines Backups auf.

**Comment by KlausStrohhaecker** [comment]
"Der Supervisor schaltet dann nach 30 sec hart ab."
Nicht der Supervisor, sondern der playground.service.

**Comment by hhoegelo** [comment]
@nonlinear-labs Der kann geschlossen werden, oder?

**Comment by nonlinear-labs** [comment]
Ja. Der ist ja noch von 2019. Da hat sich inzwischen so viel verändert, dass wir jetzt "ganz andere Probleme" haben.
